src/Holding/ViewHolding/HoldingReceivingHistory/useReceivingHistory.js

```diff
@@ -7,9 +7,9 @@ import {
 } from '@folio/stripes/core';
 import { LIMIT_MAX } from '@folio/stripes-inventory-components';
 
-
 const useReceivingHistory = (holding, tenant, options = {}) => {
   const stripes = useStripes();
+  const centralTenantId = stripes.user.user.consortium.centralTenantId;
```

Comment:
```suggestion
  const centralTenantId = stripes.user.user?.consortium?.centralTenantId;
```
lets make it optional to avoid errors when it is not consortium mode